drivers/usb/device/usb_dc_mcux_ehci.c

```diff
@@ -33,11 +37,7 @@ extern void USB_DeviceEhciIsrFunction(void *deviceHandle);
 /* The minimum value is 1 */
 #define EP_BUF_NUMOF_BLOCKS     ((NUM_OF_EP_MAX + 3) / 4)
 /* The max MPS is 1023 for FS, 1024 for HS. */
-#if defined(CONFIG_NOCACHE_MEMORY)
-__nocache K_MEM_POOL_DEFINE(ep_buf_pool, 16, 1024, EP_BUF_NUMOF_BLOCKS, 4);
-#else
-K_MEM_POOL_DEFINE(ep_buf_pool, 16, 1024, EP_BUF_NUMOF_BLOCKS, 4);
-#endif
+K_MEM_POOL_DEFINE(ep_buf_pool, 16, 2048, EP_BUF_NUMOF_BLOCKS, 4);
```

Comment:
Hi @loicpoulain There are SOC issues in the RT1050 or RT1060 that we cannot put the USB transfer buffer in cachable ram and use cache maintenance API (for example: DCACHE_InvalidateByRange) to operate USB transfer buffer. So I define the transfer buffer in the noncachable ram.  In the simple case, the issue may not be triggered, the issue may be triggered by mass bulk transfers etc. The issue is that the buffer may not updated rightly.

Reply:
It is a hardware issue which has been fixed? I do not reproduce it with RT1064 despite running in high-speed mode and stressing USB by sending DATA via ethernet over USB class. I changed the commit to take into account this issue.

Reply:
It is not fixed yet and it is hard to re-produce.

Reply:
@MarkWangChinese , coud you please reveiw again.

Reply:
It is OK, thanks